tools/src/main/scala/scala/scalanative/build/NativeConfig.scala

```diff
@@ -62,6 +62,19 @@ sealed trait NativeConfig {
   /** Map of user defined properties resolved at linktime */
   def linktimeProperties: NativeConfig.LinktimeProperites
 
+  /** Shall the resource files be embedded in the resulting binary file? Allows
+   *  the use of getClass().getResourceAsStream() on the included files. Will
+   *  not embed files with certain extensions, including ".c", ".h", ".scala"
+   *  and ".class".
+   */
+  def embedResources: Boolean
+
+  /** Base name for executable or library, typically the project name. */
+  def basename: String
+
+  /** Allow multiple main classes in the same project */
+  def multipleMains: Boolean
```

Comment:
In interactive mode, every build tool can potentially have multiple main classes: 
* `sbt` is interactive by default
* `mill` has it, here's [example how it can be used](https://alvinalexander.com/scala/mill-build-tool/how-specify-main-method-in-repl/)  
*  `scala-cli` also has an interactive mode, allowing to specify the main class when running with `-i` / `--interactive` flag

Every currently supported build tool can potentially allow multiple main classes. Requiring to explicitly allow that seems not like a good development experience.

Reply:
I wasn't aware of that. I asked someone about `mill` and got the answer that it doesn't work like `sbt` so I thought it was sbt only. Agree that it make the developer experience worse.